src/home.js

```diff
@@ -0,0 +1,37 @@
+const displayPizza = document.querySelector('.display-pizza');
+const url = 'https://forkify-api.herokuapp.com/api/v2/recipes?search=pizza&key=c9510895-3e98-4f84-a4ff-c56d29bdf5e0';
```

Comment:
the key need to be stored in another variable. like this:
const apiKey = 'c9510895-3e98-4f84-a4ff-c56d29bdf5e0'
and then create the url variable, like this:
const url = `https://forkify-api.herokuapp.com/api/v2/recipes?search=pizza&key=${apiKey}`

Reply:
the key needs to be stored in a different variable, because all of us we will need it.